Headers/Public/VLCLibrary.h

```diff
@@ -30,86 +30,79 @@
 @class VLCAudio;
 
 /**
- * The VLCLibrary is the base library of the VLCKit.framework.  This object provides a shared instance that exposes the
+ * The VLCLibrary is the base library of VLCKit.framework. This object provides a shared instance that exposes the
  * internal functionalities of libvlc and libvlc-control. The VLCLibrary object is instantiated automatically when
- * VLCKit.framework is loaded into memory.  Also, it is automatically destroyed when the VLCKit.framework is unloaded
+ * VLCKit.framework is loaded into memory.  Also, it is automatically destroyed when VLCKit.framework is unloaded
  * from memory.
  *
- * Currently, the framework does not support multiple instances of VLCLibrary.  Furthermore, you cannot destroy any
- * instiantiation of VLCLibrary, as previously noted, this is done automatically by the dynamic link loader.
+ * Currently, the framework __does not__ support multiple instances of VLCLibrary.
+ * Furthermore, you __cannot__ destroy any instance of VLCLibrary; this is done automatically by the dynamic link loader.
```

Comment:
I'm not sure if this underling syntax is permitted in doxygen. Can you verify this?